src/modules/dashboard/lists/ListItemWrapper.tsx

```diff
@@ -46,8 +51,10 @@ export const ListItemWrapper = ({
         </Link>
         {frozen && <FrozenWarning symbol={symbol} />}
         {!frozen && symbol === 'AMPL' && <AMPLWarning />}
+        {supplyCap.determineTooltipDisplay({ supplyCap })}
```

Comment:
After the frozen logic is added, this should be moved to a separate component which will determine the priority or stacked display of warnings (AMPL can be added as well):

`<ReserveWarning user={...} reserve={...} />`

Reply:
Created issue #1001